deployments/operator/samples/deviceplugin_v1_qatdeviceplugin.yaml

```diff
@@ -15,6 +15,7 @@ spec:
   kernelVfDrivers:
     - c6xxvf
     - 4xxxvf
+  provisioningConfig: qat-config
```

Comment:
I forgot to comment on this last time. I think it's not a good idea to add this by default.

Reply:
dropped